validation/src/main/scala/hmda/validation/rules/lar/syntactical/S010.scala

```diff
@@ -1,11 +1,14 @@
 package hmda.validation.rules.lar.syntactical
 
 import hmda.model.fi.lar.LoanApplicationRegister
-import hmda.validation.dsl.{ CommonDsl, Result }
+import hmda.validation.dsl.Result
 import hmda.validation.rules.EditCheck
 
 object S010 extends EditCheck[LoanApplicationRegister] {
 
+  import hmda.validation.dsl.PredicateSyntax._
+  import hmda.validation.dsl.PredicateDefaults._
+
```

Comment:
What's the reasoning behind having these imports here, rather than up with the others?  I know they are technically different scopes, but for practical purposes they seem the same, since all the code in the file is in the one object.  Having two chunks of imports feels like clutter in this case.

But I can imagine that there may be a good reason to do it.  If so, would that reason also apply to importing `Result`?